app/ui/gui.go

```diff
@@ -104,6 +104,9 @@ func runUIUbuntuTouch(e string) {
 
 	if config.Gui == "ut" {
 		cmd = exec.Command("qmlscene", "--scaling", "guis/qml/ut/MainUt.qml")
+	} else if config.Gui == "qt" {
+		guilocation := os.Getenv("AXOLOTL_GUI_DIR")
```

Comment:
If we introduce a new environment variable, could you please add it to https://github.com/nanu-c/axolotl#environment-variables ?